app/src/main/java/io/apicurio/registry/services/tenant/CustomTenantConfigResolver.java

```diff
@@ -16,41 +16,55 @@
 
 package io.apicurio.registry.services.tenant;
 
+import io.apicurio.registry.mt.TenantContext;
 import io.apicurio.registry.mt.metadata.TenantMetadataDto;
-import io.apicurio.registry.rest.Headers;
 import io.apicurio.registry.storage.RegistryStorage;
 import io.apicurio.registry.types.Current;
 import io.quarkus.oidc.OidcTenantConfig;
+import io.quarkus.oidc.OidcTenantConfig.Tls.Verification;
 import io.quarkus.oidc.TenantConfigResolver;
 import io.vertx.ext.web.RoutingContext;
 
+import java.util.Optional;
+
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
 
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
 @ApplicationScoped
 public class CustomTenantConfigResolver implements TenantConfigResolver {
```

Comment:
@Apicurio/developers - Let's be better about javadoc.  At least put something at the class level to explain what it does.  I realize that often times it's obvious what a class does when we write it.  But it may not be obvious in two years when some poor sucker needs to change it.